app/konnect/reference/konnect-api.md

```diff
@@ -97,24 +74,22 @@ via: kong/2.2.1
 {"id":"31cd0c73-g8s8-5jk7-b39b-6013salh875dd","email":"example@email.com","org_id":"fsf9agf-076b-3597-8e2d3e36ah9sfvsb","org_name":"MyOrg","first_name":"First","last_name":"Last","expiration_date":"2020-12-23T18:49:48.823Z"}%      
```

Comment:
I don't know about you, but when I try to access `https://us.api.konghq.com/kauth/api/v1/authenticate`, there is no json data in the response, only headers. Instead, it looks like this:

```
HTTP/1.1 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Expose-Headers: x-datadog-trace-id
Connection: keep-alive
Content-Length: 2
Content-Type: application/json,application/json,application/json
Via: kong/2.7.0.0-enterprise-edition
X-Kong-Proxy-Latency: 17
X-Kong-Upstream-Latency: 120
date: Fri, 22 Apr 2022 17:01:35 GMT
server: envoy
set-cookie: konnectaccesstoken=<JWT-token>; Path=/kauth/api/v1/refresh; Domain=konghq.com; Expires=Fri, 22 Apr 2022 18:01:35 GMT; HttpOnly; Secure; SameSite=Lax, KAUTH_SESSION_EXPIRY_TIME=1650648695; Path=/; Domain=konghq.com; Expires=Fri, 22 Apr 2022 18:01:35 GMT; Secure; SameSite=Lax
vary: Origin
x-b3-sampled: 1
x-b3-spanid: 768afc2248afa6f9
x-b3-traceid: 12cf381ba54a5cb6
x-datadog-span-id: 8541916867440125689
x-datadog-trace-id: 1355363704250195126
x-envoy-upstream-service-time: 119

{}
```

Can you update the response example?